### PR TITLE
Minimum requirements

### PR DESCRIPTION
**Pull request by kunicmarko20**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes
| Deprecations? | no
| Closes | #19
| License       | MIT

For now just to change requirements so #134 will pass the build and in another PR when I finish all issues then I will clean the code and add typehints/return types.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,9 +17,7 @@ matrix:
     fast_finish: true
     include:
           # Minimum supported dependencies with the latest and oldest PHP version
```
Comment (stof) [inline .travis.yml]: that comment is wrong now, as you don't have 2 jobs anymore

Comment (kunicmarko20) [reply .travis.yml]: thank you, fixed.

File: composer.json [hunk 2]
```diff
@@ -39,13 +39,13 @@
...
+        "phpunit/phpunit": "^6.0",
         "sensio/distribution-bundle": "^3.0.12 || ^4.0 || ^5.0",
-        "symfony/asset": "^2.7 || ^3.0 || ^4.0",
-        "symfony/console": "^2.7 || ^3.0 || ^4.0",
+        "symfony/asset": "^3.4 || ^4.0",
+        "symfony/console": "^3.4 || ^4.0",
         "symfony/phpunit-bridge": "^4.0",
-        "symfony/twig-bridge": "^2.7 || ^3.0 || ^4.0",
-        "symfony/yaml": "^2.7 || ^3.0 || ^4.0"
+        "symfony/twig-bridge": "^3.4 || ^4.0",
```
Comment (stof) [inline composer.json]: is twig-bridge actually optional if `twig/twig` is required ? Btw, shouldn't the required dependency be TwigBundle instead, to have the service definitions ?

Comment (kunicmarko20) [reply composer.json]: It was optional I just tested, but I agree about the TwigBundle so I added that and removed bridge from the require-dev.

**Pull request merged**